racunaljka.js

```diff
@@ -0,0 +1,131 @@
+var writes = document.getElementById('screen');
```

Comment:
Usually web applications have many modules (scripts). When you write code like here:
```
var writes = ...;
```
that is called pollution of global scope. That means that a variable which you define here will be visible in other scripts as well, since you are defining it on a global scope. In order to avoid this problem, in JS the most common solution is function scoping, where you define a function which you immediately invoke, but all variables are in its closure, so they are not polluting the global scope:
```
(function() {
  var writes = ...;
  // Rest of the module / script
})()
```